### ISSUE TITLE
Barbaric helmets don't have enchant glint

### ISSUE THREAD

**Issue by JustSkay** [MehVahdJukaar/ohmygoat#19]
On the current latest version 1.4.2 the enchantment glint is not visible. Don't know if that's always been the case or not

**Comment by MehVahdJukaar** [comment]
When enchanted you mean?

**Comment by JustSkay** [comment]
yeah. When you apply enchantments to it, the item has the glint but the model that the player wears doesn't
